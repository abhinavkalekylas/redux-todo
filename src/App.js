import "./App.css";
import { Container } from "reactstrap";
import "bootstrap/dist/css/bootstrap.min.css";

//importing components

import Todo from "./components/Todo";
import TodoForm from "./components/TodoForm";

//redux
import { Provider } from "react-redux";
import store from "./store";

const App = () => {
  return (
    <Provider store={store}>
      <Container fluid>
        <TodoForm />
        <Todo />
      </Container>
    </Provider>
  );
};

export default App;
