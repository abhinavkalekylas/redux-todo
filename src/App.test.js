import { render, screen } from "@testing-library/react";
import App from "./App";
import toJson from "enzyme-to-json";
import { shallow, mount } from "enzyme";

// test("renders learn react link", () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });

it("renders correctly App component", () => {
  const wrapper = shallow(<App />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
