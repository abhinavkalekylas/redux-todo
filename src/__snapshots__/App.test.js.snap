// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly App component 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Container
    fluid={true}
    tag="div"
  >
    <Connect(TodoForm) />
    <Connect(Todo) />
  </Container>
</Provider>
`;
